### ISSUE TITLE
GUI apps show stack trace on macOS Ventura

### ISSUE THREAD

**Issue by bjeurissen** [MRtrix3/mrtrix3#2553]
Running mrview from the terminal (installed from the binary installer) on macOS Ventura seems to work fine, but oddly produces this output in the terminal each time the window gets into focus:

```
➜  ~ mrview
1   HIToolbox                           0x00007ff81ac56726 _ZN15MenuBarInstance22EnsureAutoShowObserverEv + 102
2   HIToolbox                           0x00007ff81ac562b8 _ZN15MenuBarInstance14EnableAutoShowEv + 52
3   HIToolbox                           0x00007ff81abfa908 SetMenuBarObscured + 408
4   HIToolbox                           0x00007ff81abfa4ca _ZN13HIApplication15HandleActivatedEP14OpaqueEventRefhP15OpaqueWindowPtrh + 164
5   HIToolbox                           0x00007ff81abf4996 _ZN13HIApplication13EventObserverEjP14OpaqueEventRefPv + 252
6   HIToolbox                           0x00007ff81abbcbd2 _NotifyEventLoopObservers + 153
7   HIToolbox                           0x00007ff81abf43e6 AcquireEventFromQueue + 494
8   HIToolbox                           0x00007ff81abe35a4 ReceiveNextEventCommon + 725
9   HIToolbox                           0x00007ff81abe32b3 _BlockUntilNextEventMatchingListInModeWithFilter + 70
10  AppKit                              0x00007ff813de0f33 _DPSNextEvent + 909
11  AppKit                              0x00007ff813ddfdb4 -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 1219
12  AppKit                              0x00007ff813dd23f7 -[NSApplication run] + 586
13  libqcocoa.dylib                     0x000000010dda36aa qt_plugin_instance + 188490
14  libQt5Core.5.15.2.dylib             0x000000010e5583d7 _ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE + 471
15  libQt5Core.5.15.2.dylib             0x000000010e55c372 _ZN16QCoreApplication4execEv + 130
16  mrview                              0x000000010262bda3 _Z3runv + 83
17  mrview                              0x000000010262bafb main + 75
18  dyld                                0x0000000202a79310 start + 2432
```

Not sure what is causing this ~~, but I suspect this will be fixed once we move to qt6.~~

Other projects have reported a similar issue:

https://mail.openjdk.org/pipermail/openjfx-dev/2022-December/037642.html
https://apple.stackexchange.com/questions/452598/emacs-giving-errors-on-starting-macos-ventura

**Comment by bjeurissen** [comment]
Seems like this needs to be fixed at the side of macOS or Qt as even very mininal GUI apps experience this issue: https://issuehint.com/issue/flutter/flutter/117663

**Comment by bjeurissen** [comment]
Seems to be fixed in macOS 13.3
